modules/ROOT/pages/index.adoc

```diff
@@ -1,6 +1,2 @@
 = Creating Machine Learning Models with Red Hat OpenShift Data Science
-:navtitle: Home
-
-== Introduction
-
-This is an example quick course demonstrating the usage of Antora for authoring and publishing quick courses.
\ No newline at end of file
+:navtitle: Home
```

Comment:
Add some introductory content to this course.

You can mention the main objectives of the course, as listed in [the design doc](https://docs.google.com/document/d/1MYtVrtK88BnLb7YS-rLI39FsSlPCbDxoRN9HP75kLpk/edit).

You can use the same section in the RHODS intro course as inspiration https://redhatquickcourses.github.io/rhods-intro/rhods-intro/1.33/index.html

Reply:
Made a note of this. Will complete it in the next iteration